pkg/controller/vsphere/reconciler.go

```diff
@@ -677,6 +675,20 @@ func setProviderStatus(taskRef string, condition vspherev1.VSphereMachineProvide
 	return nil
 }
 
+func handleVsphereError(multipleFoundMsg, notFoundMsg string, defaultError, vsphereError error) error {
```

Comment:
I think elsewhere this is capitalised as `VSphere`, do we have a pattern or is it mixed everywhere?

Reply:
right, it should be `VSphere `